job_creator.php

```diff
@@ -0,0 +1,239 @@
+<?php
+
+class JobCreator
+{
+    public function getInstallerVersion($githubRepository, $branch)
+    {
+        $repo = explode('/', $githubRepository)[1];
+        if (in_array($repo, NO_INSTALLER_REPOS)) {
+            return '';
+        }
+        $v = explode('.', $branch);
```

Comment:
```suggestion
        $version = explode('.', $branch);
```
Let's please use full variable names. My small brain has trouble remembering what variables mean if they're just a single letter.